Near sputnik dao.md (#6)
near-x/learn-near-community: cn/resources/guides/NEAR-Sputnik-DAO-Empowering-Communities.md

在某种程度上，现实情况是：如果人们想要组织进入正式的小组，他们需要申请政府的许可（例如，创建公司），特别是如果他们需要银行账户并希望参加 在经济中作为一个团体。此过程耗时，昂贵，复杂，组织一旦建立限制规则就会受到限制规则约束。

即使是非正式社区协作也需要规则和审核。 想象一下，一个动漫学生组想要协调一个国际粉丝活动，或者世界各地的成功艺术家想要组成一个团体来帮助新手艺术家。社区协作可以包括通过发帖的形式解决某些问题，一起创建一些艺术或产品，一起做一个应用程序，一起投资一些数字资产，甚至像推特账户一样有个共同社交频道。 所有这些都是需要真实某种治理的例子。

很长一段时间，我喜欢努力工作的小团队，而不是营造垄断的巨大公司，然后奋斗坚持。 事实上，大约十年前，一些朋友和我创建了[Prostoksi](https://prostoksi.com/)，这是由开源项目共同努力的开发人员组成。

Web3以Web2不能的方式使这一愿景成为可能的，因为加密的保证。 想象一下，在云提供商之上或使用一组仅由一组20个人运行的银行，没有正式的联系，没有保证组织仍然在一年内运作 - 这有限制你可以建立在它上面 。 但在加密世界，基础设施和去中心化的应用程序是可靠的，因为它们是开源，开放的，并具有可靠的保证。 Apps built in this way provide common protocols to interact with each other, which other developers, businesses, and users can rely on.

DAOs是新世界此类共同协议的一部分。 自Vitalik 七年前撰写了这篇非常高水平的帖子，成千上万的成员在国内和其他网络上进行了实验，其中一些最着名的包括制造商，Moloch DAO，Metacartel，Lao和Flamingo Dao。 I’ve taken some key learnings from the first wave of DAO experimentation that informed the vision for DAOs on NEAR.

* 一般而言，社区控制治理的方式不会随着所涉及的人数扩展。 如果没有规则，法律或层次结构，成千上万的人无法高效开发或控制。 我们在协议治理和基金治理的区块链世界中看到了许多实验。 其中许多人因各种原因而失败， from technical issues to lack of interest in voting from members or lack of a unifying, driving factor。

> * 不能指望每个成员永远都会很开心。 利益变化和人们的观念改变; 多数人和少数最终会出现一些问题。 即使在大多数情况下，每个人都将通过投票等基本治理达成共识，即社区的少数群体应该能够从原来团体中分叉，并遵循新的规则, at any time if they don’t agree with the majority and have enough of their own will。

Reviewer: ```suggestion
* 不能指望每个成员永远都会很开心。 利益变化和人们的观念改变; 多数人和少数最终会出现一些问题。尽管在大多数情况下,每个人都会通过投票等基本治理达成共识,但是在任何时候,如果他们不同意多数,有足够的自己的意志  ，社区的少数群体能够从原来团体中分叉，并遵循新的规则。
```

> * 任何社区或组织都需要领导。  Even if it’s a highly dedicated group of people — if they are not working full-time on the shared cause, they are probably too busy to be active all the time and participate in every discussion or decision. Communities usually have dedicated members who are responsible for coordination, potentially moderation.

Reviewer: ```suggestion
* 任何社区或组织都需要领导。  即使这是一群高度专注的人——如果他们不是全职工作在共同的事业上，他们可能太忙了，无法一直积极参与每一个讨论或决定。 社区通常有专门的成员负责协调，潜在的缓和。
```

> * 如果一个社区是无许可和匿名， it’s still valuable to have members who are publicly known to be involved and create the “core” of the community。 社区将倾听这些成员的观点和意见，因为他们是已知最多投入和最了解情况的。

Reviewer: ```suggestion
* 如果一个社区是未经允许且匿名的，那么拥有公开参与并创建社区“核心”的成员仍然是有价值的。社区将优先听取这些成员的观点和意见，因为他们是最有投入和最有见识的人。
```

* “产品”越是去中心化形式的社区，领导力的越是重要，因为产品开发通常需要更多的背景知识，而不是任何社区参与者都有。 例如，一个小组尝试在一起创建一个区块链或去中心化交易所。
* 纯粹的基于抵押权重很多都失败可，要么是因为大多数利益相关者都无法或不想成为活动参与者，或者因为它开辟了攻击治理过程的简单方法。

> 

Reviewer: ```suggestion
Sputnik DAO是专门为技术社区而创建的吗?

- 是
- 不是

```

> 因此，DAO是一个具有统一共同目标的社区，需要一些协调和治理规则，with the flexibility to evolve over time as the interests and goals of the community shift, and the ability to transact and make money as a unit with minimal overhead。 考虑到学习这些东西，我们在NEAR创建一个DAO协议，即将授权各种社区 - 不仅仅是由技术人员组成 - 拥有灵活的，强大的框架，用于一起工作和发展。

Reviewer: ```suggestion
因此，DAO是围绕某种共同目的而团结起来的社区，需要一些协调和治理的规则，具有随着社区的利益和目标的变化而随时间发展的灵活性，以及以最小的开销作为一个单位进行交易和赚钱的能力。考虑到所有这些经验，我们开始在NEAR上创建一个DAO协议，它将赋予所有类型的社区——不仅仅是由工程师运行的技术社区——以一个灵活、健壮的框架来共同工作和发展。
```

> 

Reviewer: ```suggestion
关于社区领导，下列哪项是正确的?

-  真正的领导确保每个社区成员总是快乐的。

- 社区领导人不需要为社区全职工作。

- 即使在一个没有权限的社区中，拥有核心成员/领导者也是很重要的。

- 社区的领导人通过制定规则来管理社区。
```

> 由此产生的协议Sputnik Dao为世界各地的新的和现有社区和组织提供了一种方式，organizations around the world to have on-chain representation of their group，管理他们的链上的活动，管理自己的决策，（重要的是！）互动 与其他daos。 目前，自2021年2月的初始发布以来，大约有100个Sputnik DAO，〜500万美元分配，~$500k contained in the collective treasuries （参见这里的[统计数据](https://stats.sputnik.fund/) ）。

Reviewer: ```suggestion
由此产生的协议Sputnik Dao为世界各地的新的和现有社区和组织提供了一种方式，，可以在链上表示其组，管理其链上活动，管理其自己的决策，并(重要的是!)与其他DAO交互。自2021年2月首次发射以来，目前约有100个Sputnik dao，约50万美元已分配，约50万美元包含在国库中 （参见这里的[统计数据](https://stats.sputnik.fund/) ）。
```

查看[sputnik.fund](https://www.sputnik.fund/) 并告诉我们你的想法。下周我们将分享的UX，新功能和另一篇文章分享V2测试版，其中详细信息是如何解决组织社区的关键挑战。 确保遵循我们的[Medium](https://medium.com/sputnikdao) 收到关于V2的通知。

感谢Sputnik的早期社区组，了解V2的想法; NEAR团队，

> Tara Tan, Julia Lipton for comments and suggestions and 

Reviewer: ```suggestion
Tara Tan, Julia Lipton给予我们的意见和建议
```

> Mally Anderson  for heavy editing.

Reviewer: ```suggestion
感谢 Mally Anderson 的大量编辑
```

> Original article on Medium.

Reviewer: ```suggestion
最初的文章在Medium[Medium](https://medium.com/sputnikdao/sputnik-dao-empowering-communities-e55ac65f4433)
```

> P.S.

Reviewer: ```suggestion
顺便提及
```

> Want to contribute to LearnNEAR.Club? Submit your proposal to Learn NEAR DAO

Reviewer: ```suggestion
想为LearnNEAR.Club做贡献吗?提交您的提案学习NEAR DAO[学习NEAR DAO](https://www.sputnik.fund/#/dao/learnnear.sputnikdao.near)
```

> # NEAR Sputnik DAO:赋权的社区

Reviewer 2: ```suggetion
# NEAR Sputnik DAO: 给社区赋能
```

Open Web和Web3都源于“大众将自己构建和管理下一代互联网”的理念。 [NEAR](https://learnnear.club/what-is-near-protocol/) 以此作为使命：赋予所有人控制自己的钱，数据和治理权力。
你认为DAO的核心目的是什么？

-  协调社区活动

-  促进管理机制

> -  复制共同的架构

Reviewer 2: ```suggstion
复制企业架构
```

# NEAR Sputnik DAO: 给社区赋能
Open Web和Web3都源于“大众将自己构建和管理下一代互联网”的理念。 [NEAR](https://learnnear.club/what-is-near-protocol/) 以此作为使命：赋予所有人控制自己的钱，数据和治理权力。
你认为DAO的核心目的是什么？

-  协调社区活动

-  促进管理机制

复制企业架构

-  促进代币经济化

> 以太坊创始人Vitalik Buterin在2014年发表的一篇非常古老的博客博客帖子](https://blog.ethereum.org/2014/05/06/daos-dacs-das-and-more-an-incomplete-terminology-guide/)文章中描绘了一种思考集体行动的新方式：不是一家有股东，高管和员工的公司，而是一个去中心化的自治组织，或者是成员就是组织的一部分，贡献他们的努力，并与他们的贡献成比例地奖励。 没有老板，没有层次结构，并且参与者通过共同的目的组织在一起。

Reviewer 2: ```suggestion
以太坊创始人Vitalik Buterin在2014年发表的一篇非常古老的[博客](https://blog.ethereum.org/2014/05/06/daos-dacs-das-and-more-an-incomplete-terminology-guide/)文章中描绘了一种思考集体行动的新方式：不是一家有股东，高管和员工的公司，而是一个去中心化的自治组织，简称为DAO。其成员就是组织的一部分，贡献他们的努力，并与他们的贡献成比例地获得奖励。DAO中没有老板，没有层次结构，并且参与者通过共同的目的组织在一起。
```

> 我记得当我第一次阅读Vitalik的文章时很有同感，想象一个未来真正自治的社区，任何人的地方，任何地方都可以是一个团体，它独立于边界，法律或银行的界限 - 就像Web3一样，DAOs可以作为社区。最终公司会被DAOs替代，DAOs仍旧有很多像公司的特性，但在其成立之初以及随着相关大众的需求和优先事项的变化而具有的更大的灵活性。

Reviewer 2: ```suggestion
我记得当我第一次阅读Vitalik的文章时很有同感，想象一个未来真正自治的社区，任何地方的任何人都可以构建一个团体。它的行为不受地理界限、法规或银行体系的限制。- 就像区块链赋能的Web3给个体带来的改变，DAO同样改变了社区。最终公司会被DAO替代，仍然具有公司的特性，但在其成立之初以及随着相关大众的需求和优先事项的变化而具有的更大的灵活性。
```

> 这种愿景甚至先于我对Web3和区块链感兴趣。 我离开了乌克兰，因为我从未从那里的公司感受到安全。 我不相信任何乌克兰银行可以安全可信的存钱。 政府在过去十年中多次改变了持有美国货币的规则（比如需要把一半的钱卖给当地货币  ）。 在公司在乌克兰运营的情况下，还有很多腐败和僵化的流程。在许多国家，由于政府腐败，缺乏法治和可能消失的不靠谱的银行，造成了公司的额外开销。

Reviewer 2: ```suggestion
这种愿景甚至先于我对Web3和区块链的兴趣。 我离开了乌克兰，因为在那里建立公司让我觉得不安全。 我不相信任何乌克兰银行可以安全可信的存钱。 政府在过去十年中多次改变了持有美国货币的规则（比如需要把一半的钱转为当地货币  ）。 在乌克兰运营公司，还有很多腐败和僵化的流程。在许多国家，政府腐败，缺乏法治和可能消失的不靠谱的银行，造成了公司的额外开销。
```

> 在某种程度上，现实情况是：如果人们想要组织进入正式的小组，他们需要申请政府的许可（例如，创建公司），特别是如果他们需要银行账户并希望参加 在经济中作为一个团体。此过程耗时，昂贵，复杂，组织一旦建立限制规则就会受到限制规则约束。

Reviewer 2: ```suggestion
不止乌克兰，其他地方也类似。更进一步：如果人们想要构建一个正式组织，他们需要申请政府的许可（例如，创建公司），特别是如果他们需要开立银行账户并以组织的名义参与经济活动。此过程耗时，昂贵，复杂，组织一旦建立就面领着严格规则的约束。
```

> 即使是非正式社区协作也需要规则和审核。 想象一下，一个动漫学生组想要协调一个国际粉丝活动，或者世界各地的成功艺术家想要组成一个团体来帮助新手艺术家。社区协作可以包括通过发帖的形式解决某些问题，一起创建一些艺术或产品，一起做一个应用程序，一起投资一些数字资产，甚至像推特账户一样有个共同社交频道。 所有这些都是需要真实某种治理的例子。

Reviewer 2: ```suggestion
即使是非正式社区协作也需要规则和审核。 想象一下，一个动漫学生组想要协调一个国际粉丝活动，或者世界各地的成功艺术家想要组成一个团体来帮助新手艺术家。社区协调可以包括所有内容，从发布成员对某个问题的看法、共同创作艺术或产品、推出应用程序、投资一些数字资产，甚至共同拥有一个社交渠道（如 Twitter 帐户）。所有这些都是需要某种治理进行协调的真实例子。
```

> 很长一段时间，我喜欢努力工作的小团队，而不是营造垄断的巨大公司，然后奋斗坚持。 事实上，大约十年前，一些朋友和我创建了[Prostoksi](https://prostoksi.com/)，这是由开源项目共同努力的开发人员组成。

Reviewer 2: ```suggestion
很长一段时间以来，我相信无数小团队一起工作，而不是大公司创造垄断，然后为维护垄断而奋斗。 事实上，大约十年前，我和一些朋友创建了非公司的[Prostoksi](https://prostoksi.com/)，它由共同从事开源项目的开发人员组成。
```

> Web3以Web2不能的方式使这一愿景成为可能的，因为加密的保证。 想象一下，在云提供商之上或使用一组仅由一组20个人运行的银行，没有正式的联系，没有保证组织仍然在一年内运作 - 这有限制你可以建立在它上面 。 但在加密世界，基础设施和去中心化的应用程序是可靠的，因为它们是开源，开放的，并具有可靠的保证。以这种方式构建的应用程序提供了相互交互的通用协议，其他开发人员、企业和用户可以依赖这些协议。

Reviewer 2: ```suggestion
由于密码学带来的保证，Web3 以 Web2 无法实现的方式实现了这一愿景。 想象一下，在云提供商的基础上构建，或者使用仅由 20 人运营的银行，没有正式的隶属关系，也不能保证该组织在一年内仍能正常运作——您可以在其上构建的内容是有限的 . 但在加密领域，基础设施和去中心化应用程序是可靠的，因为它们是开源的、开放的状态，并且具有可靠的保证。 以这种方式构建的应用程序提供了相互交互的通用协议，其他开发者、企业和用户可以依赖这些协议。
```

> Web3以Web2不能的方式使这一愿景成为可能的，因为加密的保证。 想象一下，在云提供商之上或使用一组仅由一组20个人运行的银行，没有正式的联系，没有保证组织仍然在一年内运作 - 这有限制你可以建立在它上面 。 但在加密世界，基础设施和去中心化的应用程序是可靠的，因为它们是开源，开放的，并具有可靠的保证。 Apps built in this way provide common protocols to interact with each other, which other developers, businesses, and users can rely on.

Reviewer: ```suggestion
Web3以Web2不能的方式使这一愿景成为可能的，因为加密的保证。 想象一下，在云提供商之上或使用一组仅由一组20个人运行的银行，没有正式的联系，没有保证组织仍然在一年内运作 - 这有限制你可以建立在它上面 。 但在加密世界，基础设施和去中心化的应用程序是可靠的，因为它们是开源，开放的，并具有可靠的保证。以这种方式构建的应用程序提供了相互交互的通用协议，其他开发人员、企业和用户可以依赖这些协议。
```

> DAOs是新世界此类共同协议的一部分。 自Vitalik 七年前撰写了这篇非常高水平的帖子，成千上万的成员在以太网和其他网络上进行实验，其中包括一些最着名的包括制造商：Moloch DAO，Metacartel，Lao和Flamingo Dao。我从第一波DAO试验中获得了一些关键的经验，这些试验为NEAR上的DAO的远景提供了信息 。

Reviewer 2: ```suggestion
DAO 是新世界中此类通用协议的一部分。 自 Vitalik 撰写这篇非常高层次的帖子之后的七年里，成千上万的成员在以太坊和其他网络上试验了 DAO——其中一些最著名的包括 Maker、Moloch DAO、MetaCartel、The LAO 和 Flamingo DAO。 我从第一波 DAO 实验中汲取了一些关键经验，这些经验为 NEAR 上的 DAO 愿景提供了依据。
```

> *很明显，社区控制治理的简单方法不能随所涉及的人员数量而扩展  。 如果没有规则，法律或层次结构，成千上万的人无法高效开发或控制。 我们在协议治理和基金治理的区块链世界中看到了许多实验。 由于各种各样的原因，从技术问题到成员对投票缺乏兴趣，或者缺乏一个统一的驱动因素，很多都失败了。

Reviewer 2: ```suggestion
* 很明显，社区控制治理的简单方法不能随所涉及的人员数量而扩展。 如果没有规则、法律或等级制度的引入，成千上万的人就无法有效地开发或控制任何事物。 我们还在区块链领域看到了许多围绕协议和投资基金治理的实验。 其中许多都因各种原因而失败，从技术问题到成员对投票缺乏兴趣或缺乏统一的驱动因素。
```

> * 不能指望每个成员永远都会很开心。 利益变化和人们的观念改变; 多数人和少数最终会出现一些问题。尽管在大多数情况下,每个人都会通过投票等基本治理达成共识,但是在任何时候,如果他们不同意多数,有足够的自己的意志  ，社区的少数群体能够从原来团体中分叉，并遵循新的规则。

Reviewer 2: ```suggestion
不能指望每一个成员都会永远开心。 兴趣在变化，人们的意见也在变化； 多数派和少数派最终会围绕某些问题出现。 即使在大多数情况下，每个人都会通过投票等基本治理达成共识，但如果社区中的少数人不同意多数且形成了足够的自己的意愿后，他们应能随时分叉或偏离原先的群体并遵循一套新的规则。
```

* 任何社区或组织都需要领导。  即使这是一群高度专注的人——如果他们不是全职工作在共同的事业上，他们可能太忙了，无法一直积极参与每一个讨论或决定。 社区通常有专门的成员负责协调，潜在的缓和。

> * 如果一个社区是未经允许且匿名的，那么拥有公开参与并创建社区“核心”的成员仍然是有价值的。社区将优先听取这些成员的观点和意见，因为他们是最有投入和最有见识的人。

Reviewer 2: ```suggestion
* 如果一个社区是无需许可且匿名的，那么让那些知名的成员参与并创建社区的“核心”仍然很有价值。 社区将首先听取这些成员的观点和意见，因为他们被认为是最投入和最了解情况的人。。
```

> * “产品”越是去中心化形式的社区，领导力的越是重要，因为产品开发通常需要更多的背景知识，而不是任何社区参与者都有。 例如，一个小组尝试在一起创建一个区块链或去中心化交易所。

Reviewer 2: ```suggestion
这种去中心化的社区越是以“产品”为核心，就越是需要专注的领导层。因为产品开发通常需要比任何社区参与者都多得多的背景。 例如，一个社区试图共同构建区块链或去中心化交易所。
```

> * 纯粹的基于抵押权重很多都失败可，要么是因为大多数利益相关者都无法或不想成为活动参与者，或者因为它开辟了攻击治理过程的简单方法。

Reviewer 2: ```suggestion
* 纯粹基于权益的加权治理常常失败，要么是因为大多数利益相关者无法或不想成为积极的参与者，要么是因为它的治理过程容易招致攻击。
```

Sputnik DAO是专门为技术社区而创建的吗?

- 是
- 不是

因此，DAO是围绕某种共同目的而团结起来的社区，需要一些协调和治理的规则，具有随着社区的利益和目标的变化而随时间发展的灵活性，以及以最小的开销作为一个单位进行交易和赚钱的能力。考虑到所有这些经验，我们开始在NEAR上创建一个DAO协议，它将赋予所有类型的社区——不仅仅是由工程师运行的技术社区——以一个灵活、健壮的框架来共同工作和发展。

> 关于社区领导，下列哪项是正确的?

Reviewer 2: ```suggestion

关于社区领导，下列哪项是正确的?
```

-  真正的领导确保每个社区成员总是快乐的。

- 社区领导人不需要为社区全职工作。

- 即使在一个没有权限的社区中，拥有核心成员/领导者也是很重要的。

- 社区的领导人通过制定规则来管理社区。

> 由此产生的协议Sputnik Dao为世界各地的新的和现有社区和组织提供了一种方式，，可以在链上表示其组，管理其链上活动，管理其自己的决策，并(重要的是!)与其他DAO交互。自2021年2月首次发射以来，目前约有100个Sputnik dao，约50万美元已分配，约50万美元包含在国库中 （参见这里的[统计数据](https://stats.sputnik.fund/) ）。

Reviewer 2: ```suggestion

由此产生的协议 Sputnik DAO 为世界各地的新的和现有的社区和组织提供了一种方式，在链上代表他们的群体，管理其链上活动和决策，以及（重要的是！）与其他 DAO交互。 自 2021 年 2 月首次启动以来，目前约有 100 个 Sputnik DAO，分配了约 50 万美元，各个国库中总共包含约 50 万美元（参见这里的[统计数据](https://stats.sputnik.fund/) ）。
```

> 查看[sputnik.fund](https://www.sputnik.fund/) 并告诉我们你的想法。下周我们将分享的UX，新功能和另一篇文章分享V2测试版，其中详细信息是如何解决组织社区的关键挑战。 确保遵循我们的[Medium](https://medium.com/sputnikdao) 收到关于V2的通知。

Reviewer 2: ```suggestion
查看[sputnik.fund](https://www.sputnik.fund/) 并告诉我们你的想法。下周我们将分享V2 beta测试版，含有新的UX和新功能。另一篇文章将详细介绍Sputnik 如何独特地解决组织社区的关键挑战。请务必关注我们的[Medium](https://medium.com/sputnikdao) 以获取关于V2的通知。
```

> 感谢Sputnik的早期社区组，了解V2的想法; NEAR团队，

Reviewer 2: ```suggestion
感谢Sputnik的早期社区组提供V2的想法; 以及NEAR团队，
```

> Tara Tan, Julia Lipton给予我们的意见和建议

Reviewer: ```suggestion
 感谢Tara Tan, Julia Lipton给予我们的意见和建议
```

感谢 Mally Anderson 的大量编辑
最初的文章在Medium[Medium](https://medium.com/sputnikdao/sputnik-dao-empowering-communities-e55ac65f4433)
顺便提及

> 想为LearnNEAR.Club做贡献吗?提交您的提案学习NEAR DAO[学习NEAR DAO](https://www.sputnik.fund/#/dao/learnnear.sputnikdao.near)

Reviewer 2: ```suggestion
想为LearnNEAR.Club做贡献吗?提交您的提案到[学习NEAR DAO](https://www.sputnik.fund/#/dao/learnnear.sputnikdao.near)
```
